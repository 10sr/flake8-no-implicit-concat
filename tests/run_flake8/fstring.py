v = "vvv"
a = [f"aaa{v}"
     "bbb"]
b = ["ccc"
     f"ddd{v}"]
c = [f"eee{v}"
     f"fff{v}"]
d = f""
print(a, b, c, d)
