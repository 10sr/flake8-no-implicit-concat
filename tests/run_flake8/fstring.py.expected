./fstring.py:2:17: NIC001 Implicitly concatenated string literals
